assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -51,11 +53,16 @@ function useUploader() {
 			post: storyId,
 		};
 
-		return uploadMedia( file, additionalData );
+		return uploadMedia( file, additionalData ).finally( () => {
+			if ( refreshLibrary ) {
+				resetMedia();
+			}
+		});
```

Comment:
This could be written a little more cleanly with:

```js
const promise = uploadMedia( file, additionalData );
if ( ! refreshLibrary ) {
	return promise;
}
return promise.finally( resetMedia );
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/210/commits/8433e6ca8fb7d5c6f951020ec43e90be2561badd  and https://github.com/google/web-stories-wp/pull/210/commits/70a04ada1f87db933fd807671f0afddee84755a1